R/renderer.R

```diff
@@ -0,0 +1,18 @@
+#' @title Process to render number of individuals in specified states
+#'
+#' @param individualname name of the individual of interest
+#' @param statenames a list of state object's names
+#'
+#' @return rendered data
+#' @export
+#' @examples
+#' state_count_renderer_process(individual, states)
```

Comment:
This example is not runnable, which is what is failing the build. Please fix or remove

Reply:
sorting out